examples/advanced/referencing_aliasing/README.md

```diff
@@ -46,7 +46,7 @@ The reference can point to another property in a **different JSON file**. For ex
       ...
 ```
 
-It is also possible to create **chains of references**, where a value references an alias that is, in turn, another alias of a value, and so on. If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, that we just saw above is an alias of `color.base.green.value`:
+It is also possible to create **chains of references**, where a value references an alias that is, in turn, another alias of a value, and so on. If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, that we saw above is an alias of `color.base.green.value`:
```

Comment:
the end of this sentence is hard to comprehend now, should be reworded

Reply:
@chazzmoney : please suggest from below options:
1) If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, one that we saw above, is an alias of `color.base.green.value`:
2) If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, and which we saw above, is an alias of `color.base.green.value`: